src/backend/distributed/citus.sql

```diff
@@ -349,8 +378,18 @@ BEGIN
 
         -- delete partition entry
         DELETE FROM pg_dist_partition WHERE logicalrelid = v_obj.objid;
-
     END LOOP;
+
+	IF cardinality(sequence_names) = 0 THEN
+		RETURN;
+	END IF;
+
+	FOR node_name, node_port IN
+	SELECT DISTINCT name, port
+	FROM unnest(node_names, node_ports) AS nodes(name, port)
+	LOOP
+		PERFORM master_drop_sequences(sequence_names, node_name, node_port);
+	END LOOP;
 END;
 $cdbdt$;
 COMMENT ON FUNCTION citus_drop_trigger()
```

Comment:
This needs to go in an upgrade script. I's say, replace the citus.sql version with an empty function and a comment, and recreate it properly in the upgrade sql?